<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialTestWindow</class>
 <widget class="QMainWindow" name="SerialTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Serial A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkA">
        <property name="text">
         <string>Activate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="inputA"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton1">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="receiveA"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Serial B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB">
        <property name="text">
         <string>Activate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="inputB"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton2">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="receiveB"/>
    </item>
    <item>
     <widget class="QLabel" name="ctsLabel">
      <property name="text">
       <string>no CTS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuSerial_A">
     <property name="title">
      <string>Serial A</string>
     </property>
     <addaction name="actionNone"/>
    </widget>
    <widget class="QMenu" name="menuSerial_B">
     <property name="title">
      <string>Serial B</string>
     </property>
     <addaction name="actionNone_2"/>
    </widget>
    <addaction name="menuSerial_A"/>
    <addaction name="menuSerial_B"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionNone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionNone_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputA</tabstop>
  <tabstop>sendButton1</tabstop>
  <tabstop>inputB</tabstop>
  <tabstop>sendButton2</tabstop>
  <tabstop>checkA</tabstop>
  <tabstop>checkB</tabstop>
  <tabstop>receiveA</tabstop>
  <tabstop>receiveB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
